txtdirect.go

```diff
@@ -103,11 +105,25 @@ func getBaseTarget(rec record) (string, int) {
 
 func getRecord(host, path string) (record, error) {
 	zone := strings.Join([]string{basezone, host}, ".")
+	from := 0
+	if path != "" {
+		zone, from, _ = zoneFromPath(host, path)
```

Comment:
If I understood the code, zone is the variable with the new sub domain (so for `example.com/test1/test2` zone will be `test2.test1.example.com`)

I don't find the variable name very explicit ; but maybe it refers to something I'm just not familiar with :/

I also don't have a better suggestion right now, just bringing it up

Reply:
Okay apparently we already used basezone for `"_redirect"` so I see where that came from. 
Don't know why I used that name though, probably suggested by Michael.

Reply:
Zone is used in DNS quite a lot. I was more confused by "from".

Reply:
Huh, it just named it "from" because I've read this on the issue:
pkg.txtdirect.org/caddy/v1 (from=/$1/$2)
Didn't know what else to name the variable, so I used from :smile:

Reply:
So the idea behind the `from` was a simplified way to provide how a path is handled. Basically `from` what to `to`. `from` basically would be a simplified version of `re` for regex.

These names are up for discussion. Thought they made sense, but maybe not ^^.